src/util/motionModel/packaging.ts

```diff
@@ -175,6 +177,7 @@ export const packMetadata = async (
         keyframeDistance: getConfig().DX,
         resolution: '2k',
         version: '1.8',
+        deviceAnonymousId: deviceAnonymousId,
       },
```

Comment:
Also callout:
IIRC, that can break the App download/upload since App is pushing all the metadata JSON fields into its own tables too.
So releasing prior the App version that supports this new field is required.
Let me know if I'm mistaken